angular/filters/truncate_characters.js

```diff
@@ -0,0 +1,31 @@
+(function(){
+    'use strict';
+
+    angular.module('app.filters').filter('truncateCharacters', function () {
+        return function (input, chars, breakOnWord) {
+            if (isNaN(chars)) {
```

Comment:
if ( chars !== chars ) {
    return input;
}

Is that correct way to do?

Ref: https://gist.github.com/kitcambridge/1086528